### ISSUE TITLE
Mocked @method throws MethodNotFoundException.

### ISSUE THREAD

**Issue by bweston92** [phpspec/prophecy#284]
I've seen that the following has been merged #247 however I am having trouble getting it to work.

My test:
```
    /**
     * @test
     */
    public function it_will_returned_cached_version_if_set()
    {
        $redis = $this->prophesize(Client::class);
        $next = $this->prophesize(DealFinder::class);

        $expectedDealId = '6979737c-2bd0-4988-aacd-195668e3703a';
        $expectedResponse = ['deal_id' => $expectedDealId];
        
        $redis->exists(Argument::containing($expectedDealId))->willReturn(true);
        $redis->get(Argument::containing($expectedDealId))->willReturn(json_encode($expectedResponse));

        $SUT = new CachedDealFinder($redis->reveal(), $next->reveal());
        $result = $SUT->findWithId($expectedDealId);

        static::assertEquals($expectedResponse, $result);
    }
```

Produces:
```
1) CachedDealFinderTest::it_will_returned_cached_version_if_set
Prophecy\Exception\Doubler\MethodNotFoundException: Method `Double\Predis\Client\P1::exists()` is not defined.
```

Version:
```
composer info -i | grep prophecy
phpspec/prophecy v1.6.1 Highly opinionated mocking framework for PHP 5.3+
```

**Comment by ciaranmcnulty** [comment]
It looks like perhaps you're doubling the concrete `Predis\Client` class, which does not have the docblock.

Does this work if you double `Predis\ClientInterface`?

**Comment by Seldaek** [comment]
I get the same here:

```
        $redis = $this->prophesize('Predis\ClientInterface');
        $redis->hgetall('foo');
```

Results in: `Prophecy\Exception\Doubler\MethodNotFoundException: Method `Double\ClientInterface\P1::hgetall()` is not defined.` 

I am using latest prophecy (1.6.1) and tried with phpdocumentor/reflection-docblock 2.0.4 as well as 3.1.0, both fail.

**Comment by Seldaek** [comment]
Fixed by #290 + https://github.com/phpDocumentor/ReflectionDocBlock/pull/84
